red_october/red_october.py

```diff
@@ -350,16 +350,17 @@ def call(self, endpoint, method='POST', params=None, data=None):
                 success.
         """
         endpoint = '%s/%s' % (self.uri_base, endpoint)
-        if data:
-            data.update({
-                'Name': self.name,
-                'Password': self.password,
-            })
+        if data is None:
```

Comment:
Why not save two lines by setting the default of data to `{}` in the method declaration?